jsonschema/validators.py

```diff
@@ -837,18 +837,29 @@ def resolve_remote(self, uri):
         except ImportError:
             requests = None
 
+        try:
+            import yaml
```

Comment:
Can this be delayed even later so we don't add overhead if yaml is not being used?

Perhaps json importing could do the same and save some startup time.

Reply:
I do not think this is needed. A failed import is very fast, and even a successful one.